recipes/spice-hsi/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "SPICE_HSI" %}
+{% set version = "1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5660ce422871671896c8bb0084acbb22d7ff2f785e0aa49292ccc8902fb8aebd
+
+build:
+  number: 0
+  skip: true  # [py>3.7]
+
```

Comment:
I think this recipe can be noarch, https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds, so please add `noarch: python` to the build section. 
To skip python 3.8, in the `requirements` you can then add `python  <3.8`